lib/sisimai/bite/email/x1.rb

```diff
@@ -0,0 +1,124 @@
+module Sisimai::Bite::Email
+  module X1
+    # Sisimai::Bite::Email::X1 parses a bounce email which created by Unknown
+    # MTA #1. Methods in the module are called from only Sisimai::Message.
+    class << self
+      # Imported from p5-Sisimail/lib/Sisimai/Bite/Email/X1.pm
+      require 'sisimai/bite/email'
+
+      Re0 = {
```

Comment:
__[rubocop]__
* Style/MutableConstant: Freeze mutable objects assigned to constants.

<!-- Comment by [SideCI](https://sideci.com) -->